### ISSUE TITLE
There is no way to specify the audio bitrate

### ISSUE THREAD

**Issue by mcherif** [RenderHeads/UnityPlugin-AVProMovieCapture#124]
For audio output, the plugin only exposes the sampling rate and the codec order. Can we also chose the audio bitrate? This is important for cases of high fidelity audio recording, and is missing from what is exposed to the users to configure
![Capture](https://user-images.githubusercontent.com/1292828/133976066-07266e8c-fe30-46a8-8ead-d5a659eb4f8f.PNG)
.

**Comment by AndrewRH** [comment]
Yes that's correct that's currently no way to specify the bit rate.  We use a fairly high bitrate I believe 128kbps.  Otherwise you could select to use the lossless FLAC codec which would have to uncompressed quality.

Does that help?  What sort of AAC bitrates were you hoping to capture?

Thanks,
